content/en/contribute/code/workflow.md

```diff
@@ -187,6 +187,80 @@ Issues in this column are in the process of being acceptance tested by a Quality
    1. How was the issue verified? Console logs, page source, errors anywhere, execution steps.  LGTM should begin to be rarely used.
    1. Move the issue to "Ready to merge" the notify the original developer to merge the PR.
 
+Once the AT process is done, use the following templates to provide feedback in the ticket.
+
+{{< tabpane persistLang=false >}}
```

Comment:
```suggestion
{{< tabpane persistLang=false lang="markdown">}}
```
The `lang="markdown"` option will render the code block with colors according to Markdown syntax.